### PR TITLE
Gh 788 save reload results

### PR DESCRIPTION
**Pull request by p013570**

### CODE REVIEW

**Comment by p013570** [comment]
Just making a change to this.

File: ui/src/main/webapp/app/saved-results/saved-results-component.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright 2019 Crown Copyright
```
Comment (d47853) [inline ui/src/main/webapp/app/saved-results/saved-results-component.js]:
```suggestion
 * Copyright 2020 Crown Copyright
```

File: ui/src/main/webapp/app/saved-results/saved-results-component.js [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+        loading.load();
+
+        query.executeQuery(
+            {
+              "class": "uk.gov.gchq.gaffer.operation.impl.export.resultcache.GetGafferResultCacheExport",
+              "jobId": jobId
+            },
+            function(data) {
+                submitResults(data);
+            }
```
Comment (d47853) [inline ui/src/main/webapp/app/saved-results/saved-results-component.js]: May want to handle errors here (eg if the operation isn't supported)

File: ui/src/main/webapp/app/saved-results/saved-results.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
+<!--
+  ~ Copyright 2019 Crown Copyright
```
Comment (d47853) [inline ui/src/main/webapp/app/saved-results/saved-results.html]:
```suggestion
  ~ Copyright 2020 Crown Copyright
```

File: ui/src/main/webapp/app/saved-results/saved-results-component.js (hunk 2 above)
Comment (p013570) [reply ui/src/main/webapp/app/saved-results/saved-results-component.js]: The error is handled fairly well by the query.executeQuery function. The error would be something like:
Operation class uk.gov.gchq.gaffer.operation.impl.export.resultcache.GetGafferResultCacheExport is not supported by the MockAccumuloStore.

**Pull request merged**